src/components/reminders/ReminderBoard.test.tsx

```diff
@@ -0,0 +1,242 @@
+import ReminderBoard from "./ReminderBoard";
+import { useSocketContext } from "../contexts/socket/useSocketContext";
+import MockedSocket from "socket.io-mock";
+import {
+  act,
+  render,
+  screen,
+  waitFor,
+  fireEvent,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { format } from "date-fns";
+import mockdate from "mockdate";
+
+type MockedSocket = typeof MockedSocket;
+
+// Mock the socket context used to obtain the client socket
+jest.mock("../contexts/socket/useSocketContext");
+
+// Create mocked type needed by TypeScript
+const mockedUseSocketContext = jest.mocked(useSocketContext);
+
+describe("Reminder board", () => {
+  let socket: MockedSocket;
+
+  beforeEach(() => {
+    socket = new MockedSocket();
+    mockedUseSocketContext.mockReturnValue(socket.socketClient);
+    mockdate.set("2024-01-16T17:00:00.000Z");
```

Comment:
If I recall, you mentioned not actually using the mocking in your tests. if that is the case, let's delete this mocking (and the dependency) so it doesn't become a source of confusion in the future.

Reply:
I didn't end up mocking in the socket tests because it wasn't working. But I used the mocking in the ReminderBoard tests because some of the assertions of dates were off by milliseconds (I think because the time input only goes up to minutes so it seems to set the rest to zeros). I could remove the mock though and try to find a way to round the milliseconds.

Reply:
Got it. If you are using it, let's keep it. It was only if it was just hanging around unused.